### DISCUSSION TITLE
Hostname/IP does not match certificate's altnames issue

### DISCUSSION THREAD

**Discussion by mintyfusion** [vercel/next.js#22886] [Help]
We have deployed our website on Vercel and it has one API route for our customers to reach us through Contact Form. The Url is of the Azure Logic app, and we are having an issue with Hostname/IP does not match.

{"error":{"message":"request to https://prod-08.westus.logic.azure.com:443/workflows/ failed, reason: Hostname/IP does not match certificate's altnames: Host: public-web-dlcjn655u-xxxx.vercel.app. is not in the cert's altnames: DNS:westus.logic.azure.com, DNS:*.westus.logic.azure.com","type":"system","errno":"ERR_TLS_CERT_ALTNAME_INVALID","code":"ERR_TLS_CERT_ALTNAME_INVALID"}}

`export default function handler(req: NextApiRequest, res: NextApiResponse): void {
    fetch(CONTACT_API_URL, {
        method: req.method,
        body: req.body,
        headers: req.headers as HeadersInit,
    })
        .then((response: Response) => res.status(200).json(response))
        .catch((e: Error) => res.status(400).json({
            error: e,
        }));
}`

Now, the question remains how we can get passed this issue.

**Comment by amhed-dtg** [comment]
Ever figure this one out?
